test/Signature.ts

```diff
@@ -4004,124 +3521,111 @@ describe('Signature', function () {
+            await checkCreateOrderMatcherPublicKey(INPUT.data.matcherPublicKey);
+            await checkCreateOrderFee('0.04077612 TXW-DEVa4f6df');
+
+            await CommonTransaction.rejectButton.click();
+            await FinalTransactionScreen.closeButton.click();
           });
 
           it('Approved', async function () {
-            await performSignOrder.call(this, createOrder, INPUT);
+            await performSignOrder(createOrder, INPUT);
 
-            await approveMessage.call(this, 60 * 1000);
-            await closeMessage.call(this);
+            await CommonTransaction.approveButton.click();
+            await FinalTransactionScreen.closeButton.click();
 
-            const approveResult = await this.driver.executeScript<string>(
+            await browser.switchToWindow(tabOrigin);
+            const approveResult = (await browser.execute(
               () => window.result
-            );
+            )) as string;
 
             const parsedApproveResult = JSONbn.parse(approveResult);
 
             const expectedApproveResult = {
               chainId: 84,
               orderType: INPUT.data.orderType,
-              version: 4,
+              version: 4 as const,
               assetPair: {
                 amountAsset: INPUT.data.amount.assetId,
                 priceAsset: INPUT.data.price.assetId,
               },
               price: 101400200,
-              priceMode: 'fixedDecimals',
+              priceMode: 'fixedDecimals' as const,
               amount: 1000000,
               matcherFee: 4077612,
               matcherPublicKey: INPUT.data.matcherPublicKey,
               senderPublicKey,
               matcherFeeAssetId: 'EMAMLxDnv3xiz8RXg8Btj33jcEw3wLczL3JKYYmuubpc',
-            } as const;
+            };
 
             const bytes = makeOrderBytes({
               ...expectedApproveResult,
               expiration: parsedApproveResult.expiration,
               timestamp: parsedApproveResult.timestamp,
             });
 
-            expect(parsedApproveResult).to.deep.contain(expectedApproveResult);
-            expect(parsedApproveResult.id).to.equal(
-              base58Encode(blake2b(bytes))
-            );
+            expect(parsedApproveResult).toMatchObject(expectedApproveResult);
+            expect(parsedApproveResult.id).toBe(base58Encode(blake2b(bytes)));
 
             expect(
-              await verifySignature(
-                senderPublicKeyBytes,
+              verifySignature(
+                senderPublicKey,
                 bytes,
-                base58Decode(parsedApproveResult.proofs[0])
+                parsedApproveResult.proofs[0]
               )
-            ).to.be.true;
+            ).toBe(true);
           });
         });
       });
     });
 
     describe('Cancel', function () {
       const INPUT = {
-        amountAsset: '',
-        data: { id: '31EeVpTAronk95TjCHdyaveDukde4nDr9BfFpvhZ3Sap' },
-        priceAsset: '',
+        type: 1003,
```

Comment:
Previous version of `INPUT` was correct.

Reply:
done